app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -98,6 +99,7 @@ export const ContentListView: React.FC = () => {
 
   const handleRowClick = (content: IContentModel) => {
     setActiveId(content.id);
+    setContentType(content.contentTypeId);
```

Comment:
How does this work when creating new content?

Reply:
Sets to snippet when Create Snippet is clicked, and to Print when the corresponding button clicked!